components/eam/src/physics/rrtmg/radiation.F90

```diff
@@ -734,6 +740,12 @@ subroutine radiation_init(phys_state, pbuf)
                       sampling_seq='rad_lwsw', flag_xyfill=.true.)
           call addfld('FDLC'//diag(icall), (/ 'ilev' /),'I',    'W/m2', 'Longwave clear-sky downward flux', &
                       sampling_seq='rad_lwsw', flag_xyfill=.true.)
+          ! < mahf708 2023-03-21 > FNL and FNLC can be outputted:
```

Comment:
Don't add code "tags" like this (keep the descriptive comments though). Git preserves the history we need to know who did what, and when. This style of updating the code with usernames ends up adding a lot of clutter, so we actively discourage it now.